freva-rest/Dockerfile

```diff
@@ -63,14 +63,14 @@ ENV API_CONFIG=/opt/freva-rest/api_config.toml \
     # url
     API_URL=http://www.example.de/ \
     # oidc
-    OIDC_CLIENT_ID=freva \
-    OIDC_URL=http://keycloak:8080/realms/freva/.well-known/openid-configuration \
+    API_OIDC_CLIENT_ID=freva \
+    API_OIDC_URL=http://keycloak:8080/realms/freva/.well-known/openid-configuration \
     # hosts
-    REDIS_HOST=redis://localhost:6379 \
-    REDIS_SSL_CERTFILE=/certs/client-cert.pem \
-    REDIS_SSL_KEYFILE=/certs/client-key.pem \
-    SOLR_HOST=localhost:8983 \
-    MONGO_HOST=localhost:27017 \
+    API_REDIS_HOST=redis://localhost:6379 \
+    API_REDIS_SSL_CERTFILE=/certs/client-cert.pem \
+    API_REDIS_SSL_KEYFILE=/certs/client-key.pem \
+    API_SOLR_HOST=localhost:8983 \
+    API_MONGO_HOST=localhost:27017 \
```

Comment:
is it possible to remove the HOST of all services since they are not host anymore together or if you want replace it for example for redis to `REDIS_ENDPOINT`?

Reply:
May I ask why you want to use the name "endpoint"? I know we, create several services in the docker image to run most of the needed things (not Redis) in one container, but that is not a requirement. 

The PR should allow the user to choose the infrastructure. So if someone has a solr running somewhere else they should be able to use that solr, same goes for MongoDB etc. I think that would justify the name host.

But please correct me if you think I am wrong.

Reply:
it was only the naming stuff. and not only for redis. for example when I see `API_MONGO_HOST`, i expect to get `localhost ` not along with a port. but do whatever u think is alright, not very crucial